Circuit_Playground_Express_cpx_Library_Examples/cpx_touch_tone_machine.py

```diff
@@ -0,0 +1,29 @@
+"""This example requires seven wav files from the Additional_Content folder on the PyCon2019 GitHub
+repo. Download the files and copy them to your CIRCUITPY drive. Once copied, try touching the touch
+pads on A1-A7 to hear a funky scale of notes!"""
```

Comment:
The "I had to memorize all the major and minor scales" part of my brain hates this but it works great!